cf_xarray/accessor.py

```diff
@@ -1482,6 +1482,91 @@ def stack(self, dimensions=None, **dimensions_kwargs):
             dimensions.update({key: tuple(itertools.chain(*updates))})
         return self._obj.stack(dimensions)
 
+    def differentiate(
+        self,
+        coord,
+        edge_order: int = 1,
+        datetime_unit: str = None,
+        follow_positive: bool = False,
```

Comment:
maybe we need a different kwarg name. This option makes it so that positive is always up like in a right-handed coordinate system. Ideas?

Reply:
I can't think of any good alternative. Maybe we just document the coordinate system assumed by `cf_xarray`?